### PR TITLE
Uninstall-Package and Unregister-PackageSource - style, copyedits, fixed placeholders

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

- Replaced placeholders with content
- Style and copyedits

Uninstall-Package Acrolinx scores
Original version: 89
Updated version: 97

Unregister-PackageSource Acrolinx scores
Original version: 87
Updated version: 97

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[b7adc3d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4360/commits/b7adc3d2493d7554bbdf719f1977a2fe667abe6f)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 17 sec(s)_, this is based on the last _55_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[b7adc3d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4360/commits/b7adc3d2493d7554bbdf719f1977a2fe667abe6f)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.0/PackageManagement/Unregister-PackageSource.md) | :x:Error | | [Details](#user-content-3284143f-3a3a7cff)
[reference/5.1/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.1/PackageManagement/Unregister-PackageSource.md) | :x:Error | | [Details](#user-content-0cf8255b-3a3a7cff)
[reference/6/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/6/PackageManagement/Unregister-PackageSource.md) | :x:Error | | [Details](#user-content-a2e57d1b-3a3a7cff)
[]() | :x:Error | | [Details](#user-content-3a3a7cff)

<a id="3284143f-3a3a7cff"></a>
### [reference/5.0/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.0/PackageManagement/Unregister-PackageSource.md)
 - **[Error]** 
`````
Failed to parse 5.0/PackageManagement/Unregister-PackageSource.md due to :307:(1306) '

[Set-PackageSource](Set-PackageSource.md'
 Expect hyperlink, but got [Set-PackageSource](Set-PackageSource.md, please use PlatyPS to do content validation
`````

<a id="0cf8255b-3a3a7cff"></a>
### [reference/5.1/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.1/PackageManagement/Unregister-PackageSource.md)
 - **[Error]** 
`````
Failed to parse 5.1/PackageManagement/Unregister-PackageSource.md due to :368:(1306) '

[Set-PackageSource](Set-PackageSource.md'
 Expect hyperlink, but got [Set-PackageSource](Set-PackageSource.md, please use PlatyPS to do content validation
`````

<a id="a2e57d1b-3a3a7cff"></a>
### [reference/6/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/6/PackageManagement/Unregister-PackageSource.md)
 - **[Error]** 
`````
Failed to parse 6/PackageManagement/Unregister-PackageSource.md due to :368:(1306) '

[Set-PackageSource](Set-PackageSource.md'
 Expect hyperlink, but got [Set-PackageSource](Set-PackageSource.md, please use PlatyPS to do content validation
`````

<a id="3a3a7cff"></a>
###
 - **Line 138**: **[Error]** `````W:\wels-s\.optemp\packages\opbuild.scripts.1.75.0\tools\opbuild\docset.prebuild.ps1 failed to execute`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905281545350194-4360%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=vTBr5i1Qz%2Bm8BJ8C0qFWN7%2BIPgAW%2Bx3CakQe%2FKM7G9g%3D&st=2019-05-28T15%3A51%3A50Z&se=2019-06-28T15%3A56%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[8ed249d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4360/commits/8ed249db3153ab2b6ea680207dd33b74a52a5629)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 17 sec(s)_, this is based on the last _55_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[8ed249d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4360/commits/8ed249db3153ab2b6ea680207dd33b74a52a5629)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PackageManagement/Uninstall-Package.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.0/PackageManagement/Uninstall-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/uninstall-package?view=powershell-5.0&branch=pr-en-us-4360) (powershell-5.0) |
[reference/5.0/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.0/PackageManagement/Unregister-PackageSource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/unregister-packagesource?view=powershell-5.0&branch=pr-en-us-4360) (powershell-5.0) |
[reference/5.1/PackageManagement/Uninstall-Package.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.1/PackageManagement/Uninstall-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/uninstall-package?view=powershell-5.1&branch=pr-en-us-4360) (powershell-5.1) |
[reference/5.1/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.1/PackageManagement/Unregister-PackageSource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/unregister-packagesource?view=powershell-5.1&branch=pr-en-us-4360) (powershell-5.1) |
[reference/6/PackageManagement/Uninstall-Package.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/6/PackageManagement/Uninstall-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/uninstall-package?view=powershell-6&branch=pr-en-us-4360) (powershell-6) |
[reference/6/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/6/PackageManagement/Unregister-PackageSource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/unregister-packagesource?view=powershell-6&branch=pr-en-us-4360) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905281613403027-4360%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=77pZWtmSQvqqEk4v7loazg3EySxorfTq%2FXvvZoMG8dE%3D&st=2019-05-28T16%3A20%3A04Z&se=2019-06-28T16%3A25%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/PackageManagement/Uninstall-Package.md [hunk 1]
```diff
@@ -213,7 +225,9 @@ Accept wildcard characters: False
 
 ### -InputObject
 
-Specifies a package by using the package's SoftwareIdentity type, which is shown in the results of the Get-Package cmdlet.
+Accepts pipeline input that specifies the package's **SoftwareIdentity** object from the
+`Get-Package` dmdlet. **InputObject** accepts the **SoftwareIdentity** object as a `Get-Package`
```
Comment (sdwheeler) [inline reference/5.0/PackageManagement/Uninstall-Package.md]: Typo: dmdlet

Comment (davidsmatlak) [reply reference/5.0/PackageManagement/Uninstall-Package.md]: Thanks! Fixed and uploaded changed files.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[14c178d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4360/commits/14c178dfb6deacd75d90ef744e43ec615418b9be)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 17 sec(s)_, this is based on the last _57_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[14c178d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4360/commits/14c178dfb6deacd75d90ef744e43ec615418b9be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PackageManagement/Uninstall-Package.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.0/PackageManagement/Uninstall-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/uninstall-package?view=powershell-5.0&branch=pr-en-us-4360) (powershell-5.0) |
[reference/5.0/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.0/PackageManagement/Unregister-PackageSource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/unregister-packagesource?view=powershell-5.0&branch=pr-en-us-4360) (powershell-5.0) |
[reference/5.1/PackageManagement/Uninstall-Package.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.1/PackageManagement/Uninstall-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/uninstall-package?view=powershell-5.1&branch=pr-en-us-4360) (powershell-5.1) |
[reference/5.1/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/5.1/PackageManagement/Unregister-PackageSource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/unregister-packagesource?view=powershell-5.1&branch=pr-en-us-4360) (powershell-5.1) |
[reference/6/PackageManagement/Uninstall-Package.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/6/PackageManagement/Uninstall-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/uninstall-package?view=powershell-6&branch=pr-en-us-4360) (powershell-6) |
[reference/6/PackageManagement/Unregister-PackageSource.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-pkgmgmt3740/reference/6/PackageManagement/Unregister-PackageSource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/packagemanagement/unregister-packagesource?view=powershell-6&branch=pr-en-us-4360) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905281732565708-4360%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=B3d07R8QJIWiLCcOu5f97BlLwav7EcKG5dERyGfhBpc%3D&st=2019-05-28T17%3A39%3A17Z&se=2019-06-28T17%3A44%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**